apollos-church-api/config.yml

```diff
@@ -243,7 +243,7 @@ DISCOVER_FEATURES:
         arguments:
           id: '{"search":"messages_production_trending","name":"Crowd Favorites","filters":"ministries:\"The Porch\""}'
           limit: 3
-    type: HorizontalCardList
+    type: ActionList
     subtitle: Crowd Favorites
     primaryAction:
```

Comment:
We should probably remove the `primaryAction` from here, as it doesn't apply to the HorizontalList.

Reply:
Nvm, looking at the wrong block.